### ISSUE TITLE
error in example file

### ISSUE THREAD

**Issue by ksdavidc** [zjhcn/obsidian-code-preview#6]
1. link to example file is wrong
2. regex example is missing /s:

```
path: /sub/color.css
start: body
end: dd\\d{2}
```
should be
```
path: /sub/color.css
start: body
end: /dd\d{2}/
```

**Comment by hankjs** [comment]
Thanks for using plugin! Fixed in version [1.3.11](https://github.com/zjhcn/obsidian-code-preview/releases/tag/1.3.11).

**Comment by ksdavidc** [comment]
You're welcome!
